elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/SQLQueryEngine.java

```diff
@@ -55,9 +55,9 @@
     @Getter
     private Map<Class<?>, Table> tables;
 
-    public SQLQueryEngine(EntityManagerFactory emf, EntityDictionary dictionary, MetaDataStore metaDataStore) {
+    public SQLQueryEngine(EntityManagerFactory emf, MetaDataStore metaDataStore) {
         this.emf = emf;
-        this.dictionary = dictionary;
+        this.dictionary = metaDataStore.getDictionary();
```

Comment:
Can we change `dictionary` to `metaDataDictionary` as this dictionary is not the exposed Elide core dictionary?